src/jit/liveness.cpp

```diff
@@ -1323,6 +1322,13 @@ class LiveVarAnalysis
 
 void Compiler::fgLiveVarAnalysis(bool updateInternalOnly)
 {
+#if CAN_DISABLE_DFA
```

Comment:
Yeah, I was considering adding a parameter to `fgLocalVarLiveness` to communicate that info.